src/main/java/com/develhope/spring/order/repository/OrderRepository.java

```diff
@@ -1,9 +1,13 @@
 package com.develhope.spring.order.repository;
 
 import com.develhope.spring.order.entity.Order;
+import com.develhope.spring.vehicles.entity.Vehicle;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
 @Repository
 public interface OrderRepository extends JpaRepository <Order, Long> {
+    List<Order> findByUserId(Long userId);
```

Comment:
il metodo va bene ma sarebbe ancora meglio uniformarlo alla libreria che abbiamo usato per creare i filtri in Vehicle (VehicleServiceFilter) e creare una query di quel tipo. In ogni caso va bene anche questo